### ISSUE TITLE
Use d3 transitions when updating plots

### ISSUE THREAD

**Issue by andredumas** [andredumas/techan.js#4]
Would be nicer if d3 transitions was used to update data: [Data Feed Example](http://bl.ocks.org/andredumas/95f1f22130fb1a3a8181)

**Comment by prionkor** [comment]
Would like to use transitions when updating chart :+1:

**Comment by andredumas** [comment]
Yes it would be very nice. Some of the updates scheduled for 0.6.0 should bring this a little closer.

**Comment by gleam-ru** [comment]
+1
when v-0.6 will be released?

**Comment by andredumas** [comment]
@gleam-ru I'm not sure exactly when, but I will be pushing changes for this soon. Unfortunately my day job has been very busy the last few months.

**Comment by adamrabie** [comment]
+1
